homotopy-web/src/app/signature/item.rs

```diff
@@ -146,7 +146,10 @@ struct CustomRecolorButtonProps {
 
 #[function_component(CustomRecolorButton)]
 fn custom_recolor_button(props: &CustomRecolorButtonProps) -> Html {
-    let hex = format!("#{:X}", props.value.0);
+    let hex = {
```

Comment:
This could also use the `hex` function on `Color` that I suggested in a comment above.